content/tutorial/tutorial.md

```diff
@@ -976,30 +983,30 @@ Finally, we need to move the `handleClick` method from the Board component to th
   }
 ```
 
->Note
+>ध्यान दें
 >
->Unlike the array `push()` method you might be more familiar with, the `concat()` method doesn't mutate the original array, so we prefer it.
+>Array के विपरीत `push()` मेथड से आप अधिक परिचित हो सकते हैं, `concat()` मेथड मूल array को म्यूटेट नहीं करती है, इसलिए हम इसे पसंद करते हैं।
```

Comment:
incorrect translation, changes meaning of sentence

Reply:
```
Array के `push()` मेथड से आप अधिक परिचित हो सकते हैं, `concat()` मेथड मूल array को म्यूटेट नहीं करती है, इसलिए हम इसे पसंद करते हैं।
```

sounds fine to me, how about you?

Reply:
mutate nai karta hai, rest is fine